Turkey/TestOutputFormat.cs

```diff
@@ -63,11 +63,14 @@ public async override Task AfterRunningTestAsync(string name, TestResult result,
 
             public async override Task PrintFailedTests()
             {
-                Console.WriteLine();
-                Console.WriteLine("The following tests failed: ");
-                foreach(var test in failedTests)
+                if (failedTests.Count != 0)
                 {
-                   Console.WriteLine($"{string.Format("{0,-30}", test.Name)}({test.Duration})");
+                    Console.WriteLine();
```

Comment:
May be we shouldn't call `PrintFailedTests()` when there are no failed tests?

Reply:
Or we could also remove `PrintFailedTests` and move this into `AfterRunningAllTestsAsync`.

Reply:
@tmds I rewrote it to not call ```PrintFailedTests``` if there are no failed tests.